### ISSUE TITLE
TalkBack select hidden items

### ISSUE THREAD

**Issue by ericfs** [ampproject/amphtml#28293]
## What's the issue?

Items that are not visible are selectable using TalkBack gesture navigation to cycle through items.

![Screenshot_20200508-165029](https://user-images.githubusercontent.com/851850/81452559-2d828e80-9155-11ea-9904-856ee7566210.png)
![Screenshot_20200508-165010](https://user-images.githubusercontent.com/851850/81452527-19d72800-9155-11ea-92d7-935f3c7498ed.png)

## How do we reproduce the issue?

1. Enable TalkBack
2. Open a story such as https://www.10best.com/web-stories/amp/what-are-jonnycakes-and-why-do-rhode-islanders-love-them/
3. Use swipe gesture to cycle through items

## What browsers are affected?

Android Chrome

## Which AMP version is affected?

2004252135000

**Comment by gmajoulet** [comment]
Closed by #28405

cc @processprocess
